### DISCUSSION TITLE
[recommended] modify the default deployment password

### DISCUSSION THREAD

**Discussion by HantaoCai** [nocodb/nocodb#2861] [General]
It is recommended that developers modify the default deployment password 'password', we were extorted by btc to restore the database......

![image](https://user-images.githubusercontent.com/44667064/181403808-5b0f70f0-381c-4c4a-a9c5-45aa4f5f839f.png)

**Comment by youyiio** [comment]
unbelievable!

**Comment by wingkwong** [comment]
Could you give more context on this? What do you mean by `default deployment password` ? Like `password` in the below the command?

```bash
docker run -d --name nocodb-mysql \
-v "$(pwd)"/nocodb:/usr/app/data/ \
-p 8080:8080 \
-e NC_DB="mysql2://host.docker.internal:3306?u=root&p=password&d=d1" \
-e NC_AUTH_JWT_SECRET="569a1821-0a93-45e8-87ab-eb857f20a010" \
nocodb/nocodb:latest
```

**Comment by HantaoCai** [comment]
I mean you should replace the password on the template with the new mysql account password.
Don't be lazy ..

I started with docker-compose and didn't modify any configuration😌

**Comment by o1lab** [comment]
Which docker-compose did you try ? We do not have db port exposed to out side. Only the server port 8080.

**Comment by o1lab** [comment]
Moving this to discussion :D
